kalite/distributed/templates/distributed/base.html

```diff
@@ -168,9 +168,10 @@
                           <ul class="nav navbar-nav navbar-right nav-tabs top-nav" role="tablist">
                               <div class="visible-xs">
                                   <li id="search_input">
-                                      <form class="navbar-form " action="{% url 'search' %}" method="get"  role="search">
+									  <form class="navbar-form " action="{% url 'search' %}" method="get"  role="search">
                                           {% comment %}translators: this will appear in the navigation bar. please try to keep it as short as possible.{% endcomment %}
-                                          <input type="text" name="query" id="search_box" class="form-control pull-left" placeholder="{% trans 'Topic, video, exercise...' %}" />
+                                          <p class="search-label search-label-mini"><label for="search_box">Search</label></p>
```

Comment:
I am concerned how the addition of text will affect layout, could you post a screenshot of how this renders?

Also include screenshots for smaller form factors - if you are unfamiliar with how to do this, here are instructions for doing so in Chrome: https://developer.chrome.com/devtools/docs/device-mode